### ISSUE TITLE
[TSDoc> Building the project]: Ensure links are distinguished from surrounding text in a way that does not rely on color (a[href$="rushjs.io"])

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#266]
**Test Environment:**
Version: Version 24H2 (OS Build 26257.5000)
Browser Version: 129.0.2779.0 (Official build) dev (64-bit)
URL: https://tsdoc.org/pages/contributing/building/
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on EDGE Dev browser.
2. Run the tool and observe the issue.

**Actual Result:**
Ensures links have distinguished from surrounding text in a way that does not rely on color. Currently it is 1.77:1 which is less than 3:1.

**Expected Result:**
Color contrast ratio for the link with surrounding text should be equal to or greater than 3:1.

**Element path:** 
a[href$="rushjs.io"]

**Snippet:**
 <a href="https://rushjs.io" target="_blank" rel="noopener noreferrer">https://rushjs.io</a>

**How to fix:** 
Fix any of the following:
  The link has insufficient color contrast of 1.77:1 with the surrounding text. (Minimum contrast is 3:1, link text: #008028, surrounding text: #444950)
  The link has no styling (such as underline) to distinguish it from the surrounding text

**User Impact:**
It will impact the low vision as they will not able to distinguish the link with surrounding text.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

![Ensures links have distinguished from surrounding text in a way that does not rely on color](https://github.com/user-attachments/assets/c96a9ee2-1d57-49b3-8745-4116ac7de5cd)

![Luminosity ratio for the link text is less than 3 1](https://github.com/user-attachments/assets/15e51ec7-c0f3-4c5c-90a6-bbb7a3fc915f)
